subsys/bluetooth/controller/ll_sw/openisa/lll/lll.c

```diff
@@ -166,6 +166,20 @@ int lll_init(void)
 	return 0;
 }
 
+int lll_deinit(void)
+{
+	/* Disable IRQs */
+	irq_disable(LL_RADIO_IRQn);
+	irq_disable(LL_RTC0_IRQn);
+	irq_disable(HAL_SWI_RADIO_IRQ);
+#if defined(CONFIG_BT_CTLR_LOW_LAT) || \
```

Comment:
Use IS_ENABLE here too? (I have not checked if these are the correct IRQs for openisa port...)

Reply:
Fixed